src/markdown-pages/docs/running-simbank-tests/running-simbank-tests-cli.md

```diff
@@ -50,7 +50,7 @@ In order to run the Galasa SimBanks tests you need to add some configuration inf
 
 The SimBank tests are held in the <a href="https://github.com/galasa-dev/simplatform" target="_blank"> Galasa simplatform repository</a> in GitHub. To start running the tests you need to clone the repository, if you have not already done so. To find out how to clone the cli repository, follow the instruction in the [Running Galasa SimBank online](../running-simbank-tests/simbank-cli) documentation.
 
-After cloning the repository, complete the following steps to run the SimBankIVT test that is provided with Galasa. The following example uses SimBank OBR version `0.25.0` and Galasa uber OBR version `0.37.0`.
+After cloning the repository, complete the following steps to run the SimBankIVT test that is provided with Galasa. The following example uses SimBank OBR version `0.25.0` and Galasa uber OBR version `0.38.0`.
```

Comment:
I think the simbank obr was bumped also ? @jadecarino ?

Reply:
Jade updated that in #843 

https://github.com/galasa-dev/galasa.dev/blob/next/src/markdown-pages/docs/running-simbank-tests/running-simbank-tests-cli.md?plain=1#L53